src/jit/compiler.cpp

```diff
@@ -4713,165 +4713,165 @@ unsigned getMethodBodyChecksum(__in_z char *code, int size)
 }
 
 
-int           Compiler::compCompileHelper (CORINFO_MODULE_HANDLE            classPtr,
-                                           COMP_HANDLE                      compHnd,
-                                           CORINFO_METHOD_INFO            * methodInfo,
-                                           void *                         * methodCodePtr,
-                                           ULONG                          * methodCodeSize,
-                                           CORJIT_FLAGS                   * compileFlags,
-                                           CorInfoInstantiationVerification instVerInfo)
-    {
-        CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
+int           Compiler::compCompileHelper(CORINFO_MODULE_HANDLE            classPtr,
+                                          COMP_HANDLE                      compHnd,
+                                          CORINFO_METHOD_INFO            * methodInfo,
+                                          void *                         * methodCodePtr,
+                                          ULONG                          * methodCodeSize,
+                                          CORJIT_FLAGS                   * compileFlags,
+                                          CorInfoInstantiationVerification instVerInfo)
+{
+    CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
```

Comment:
The problem with massive changes is that they clutter the PR review as well as the source history. When they are mixed with actual code changes it makes it that much harder to review the code, and to understand the code changes later. I don't think it is useful to make reformatting changes unless the code is being brought up to CC standards.